### PR TITLE
 adding index module code examples

### PR DESCRIPTION
**Pull request by anniegale9538**
Hi! I found additional docs that do not have the Python code examples.

[Index Modules](https://www.elastic.co/guide/en/elasticsearch/reference/master/index-modules.html):

- [Index-level shard allocation filtering](https://www.elastic.co/guide/en/elasticsearch/reference/master/shard-allocation-filtering.html)
- [Delaying allocation when a node leaves](https://www.elastic.co/guide/en/elasticsearch/reference/master/delayed-allocation.html)
- [Index recovery prioritization](https://www.elastic.co/guide/en/elasticsearch/reference/master/recovery-prioritization.html)
- [Index-level data tier allocation filtering](https://www.elastic.co/guide/en/elasticsearch/reference/master/allocation-total-shards.html)
- [Index blocks](https://www.elastic.co/guide/en/elasticsearch/reference/master/index-modules-blocks.html)
- [Similarity module](https://www.elastic.co/guide/en/elasticsearch/reference/master/index-modules-similarity.html)
- [Slow Log](https://www.elastic.co/guide/en/elasticsearch/reference/master/index-modules-slowlog.html)
- [Store](https://www.elastic.co/guide/en/elasticsearch/reference/master/index-modules-store.html)
- [Preloading data into the file system cache](https://www.elastic.co/guide/en/elasticsearch/reference/master/preload-data-to-file-system-cache.html) ** part of the Store ascii doc
- [Index Sorting](https://www.elastic.co/guide/en/elasticsearch/reference/master/index-modules-index-sorting.html)
- [Indexing pressure](https://www.elastic.co/guide/en/elasticsearch/reference/master/index-modules-indexing-pressure.html)

### CODE REVIEW

File: utils/generate-examples.py [hunk 1]
```diff
@@ -143,6 +143,17 @@
...
+    "index-modules/allocation/delayed.asciidoc",
+    "index-modules/allocation/prioritization.asciidoc",
+    "index-modules/allocation/total_shards.asciidoc",
+    "index-modules/allocation/data_tier_allocation.asciidoc",
+    "index-modules/blocks.asciidoc",
+    "index-modules/similarity.asciidoc",
+    "index-modules/slowlog.asciidoc",
+    "index-modules/store.asciidoc",
+    "index-modules/index-sorting.asciidoc",
+    "index-modules/indexing-pressure.asciidoc"
```
Comment (pquentin) [inline utils/generate-examples.py:156]: Would you mind adding a trailing comma here to fix the lint!

**Review by pquentin — CHANGES_REQUESTED** [review]
Thanks! That's great. Sorry for the delay, I was on PTO.

File: utils/generate-examples.py (hunk 1 above)
Comment (anniegale9538) [reply utils/generate-examples.py]: I hope you had the best time off! I have been careful to only add the examples I have investigated, but noticed a number for pre-existing files that look to be making some changes to their examples, I was going to see if I pushed a draft if we could review them and confirm that they should be added, but wanted to get a feel for it this is over stepping.

Comment (pquentin) [reply utils/generate-examples.py]: Hey Annie, thanks for the careful consideration here. This is not over stepping, but maybe unnecessary work as we update existing examples just before releases anyway.

**Review by pquentin — APPROVED** [review]
Thanks for iterating! LGTM.

File: utils/generate-examples.py (hunk 1 above)
Comment (pquentin) [reply utils/generate-examples.py]: What would be nice however would be to modify utils/generate-examples.py to exclude empty examples.

Comment (anniegale9538) [reply utils/generate-examples.py]: I know I way behind on this but would love to take a whack at modifying utils/generate-examples.py to exclude empty examples, if you haven't already?

**Pull request merged**